lib/auth/machineid/machineidv1/bot_service.go

```diff
@@ -612,6 +621,22 @@ func botFromUserAndRole(user types.User, role types.Role) (*pb.Bot, error) {
 		},
 	}
 
+	// Copy in labels from the user
+	b.Metadata.Labels = map[string]string{}
+	for k, v := range user.GetMetadata().Labels {
+		// We exclude the two labels that are implicitly added to the user by
+		// the bot service.
+		specialLabels := []string{
+			types.BotLabel,
+			types.BotGenerationLabel,
+		}
+		if slices.Contains(specialLabels, k) {
```

Comment:
Sure - I can move the slice declaration out of the loop. I generally prefer using `slices.Contains` versus `==` since it lets me give a more descriptive name to what I am comparing with.

Reply:
b047e891cb